resdk/tests/unit/test_resolwequerry.py

```diff
@@ -4,19 +4,25 @@
 # pylint: disable=missing-docstring, protected-access
 
 import unittest
+
 import six
+import slumber
 
 from mock import patch, MagicMock
 
 from resdk.resolwe import ResolweQuerry
 
-if six.PY2:
-    # pylint: disable=deprecated-method
-    unittest.TestCase.assertRegex = unittest.TestCase.assertRegexpMatches
-
 
 class TestResolweQuerry(unittest.TestCase):
 
+    @patch('resdk.resolwe.getattr')
+    @patch('resdk.resolwe.Resolwe')
+    @patch('resdk.resources.sample.Sample')
```

Comment:
Do we really wish to patch everything? I would not patch parts of our SDK, but calls to external functions (if necessary). If you would not patch Sample here, you would cover some part of the Sample class as well. Is this bad? @dblenkus, what do you think?